package.json

```diff
@@ -37,7 +37,6 @@
     "acorn-walk": "^8.0.0",
     "chalk": "^4.1.0",
```

Comment:
Can be replaced by `colorette` (light, fast and with simple API), already used in `webpack-cli@4`, so for developers which use `webpack-cli` and `webpack-bundle-analyzer` we will have less deps

Reply:
Good point but something for a different PR :)